internal/pkg/service/buffer/sink/tablesink/storage/config.go

```diff
@@ -4,28 +4,40 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage/level/local"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage/level/staging"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage/level/target"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage/volume/assignment"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage/volume/registration"
 )
 
 // Config contains default configuration for the storage.
 // Default settings can be customized in definition.TableSink.
 type Config struct {
-	Local   local.Config   `configKey:"local"`
-	Staging staging.Config `configKey:"staging"`
-	Target  target.Config  `configKey:"target"`
+	VolumeAssignment   assignment.Config   `json:"volumeAssignment"`
+	VolumeRegistration registration.Config `configKey:"volumeRegistration"`
```

Comment:
`storage.Config` now contains assignment and registratons configurations.
Before the modification, they were part of the `local.Config`.